index.html

```diff
@@ -2041,10 +2031,11 @@ <h2>The <dfn>record type</dfn> string</h2>
     The serial number usually consists of 4 or 7 numbers, separated by `:`.
   </p>
   <p>
-    The <dfn>NDEFWriter</dfn> is an object used for writing data to an <a>NFC tag</a>.
+    The <dfn>NDEFReader</dfn> is an object used for reading or writing data when
```

Comment:
I believe this is redundant with the first paragraph of "The NDEFReader object"

Reply:
Removed.